apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageDeletor.java

```diff
@@ -33,18 +30,7 @@ public PageDeleteResponse deletePageDraft(Long id) {
     }
 
     if (page.getTemplateType().equals(PageConstants.DRAFT)) {
-
-      Long publishedWithDraft = pageUidFinder.findTemplateByType(page.getFormCd(), PageConstants.PUBLISHED_WITH_DRAFT);
-
-      WaTemplate publishedWithDraftPage = entityManager.find(WaTemplate.class, publishedWithDraft);
-
-      if (publishedWithDraftPage == null) {
-        throw new PageNotFoundException();
-      }
-      publishedWithDraftPage.setTemplateType(PageConstants.PUBLISHED);
-
       entityManager.remove(page);
-
```

Comment:
Question: If there is a `Published with Draft` version of this page, won't it be stuck as `Published with Draft` instead of being set to `Published` after these changes?

Reply:
hmm...yes, I'll fix this now.